### ISSUE TITLE
Loads fine when the page is browsed too directly, however problem with iron router?

### ISSUE THREAD

**Issue by ChrisJGayle** [MaazAli/Meteor-Malihu-Custom-Scrollbar#3]
HTML CODE:
<template name = 'content'>
  <div id = 'content'>
   {{> yield}}
  </div>
</template>

JS CODE:
Template.content.onRendered(function () {
  $("#content").mCustomScrollbar({theme:"my-theme"});
  console.log('Scrollbar initialized.');
});

CONSOLE:
Scrollbar initialized.
Exception from Tracker recompute function: undefined
Error: Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted is not a child of this node.

-
If I use the browse bar and hit enter to navigate to a page, everything works fine.

Whenever I click to navigate through the application using these links; 

<a href="{{pathFor 'signup'}}"> <div id = 'signup'>
  <i class="add user icon"></i>
</div> </a>

I am met with the above error, also nothing is shown at all as it should with {{> yield}} 

Has anyone experienced this? Or able to tell me where I have went wrong?

**Comment by sveinburne** [comment]
+1, have got the same problem.

**Comment by MaazAli** [comment]
Are you guys getting this if you remove this addon? It's very likely that this addon is now out of date.

**Comment by sveinburne** [comment]
Yes I can confirm you, happens with the addon. I need to have the {{> yield}} inside my layoutTemplate (declared in Router.configure()) to avoid the problem... but have no idea why.
